ui/src/lib/gql/provider/GraphQLRequestMiddleware.ts

```
@@ -0,0 +1,200 @@
+//     return new Blob(chunks);
+//   }
+
+//   return new Blob([data]);
+// };
+
+const isExtractableFile = <ValueType>(value: ValueType) => {
+  return (
+    (typeof File !== "undefined" && value instanceof File) ||
+    (typeof Blob !== "undefined" && value instanceof Blob) ||
+    (typeof Buffer !== "undefined" && value instanceof Buffer) ||
+    (typeof value === `object` &&
```

Biome

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)